Makefile

```diff
@@ -462,7 +462,13 @@ else
 endif
 .PHONY: fmt/shfmt
 
-lint: lint/shellcheck lint/go lint/ts lint/examples lint/helm lint/site-icons lint/markdown
+fmt/docs-md-tables:
+	cd site
+	../scripts/pnpm_install.sh
+	pnpm exec markdown-table-formatter "../docs/**/*.md"
+.PHONY: fmt/docs-md-tables
```

Comment:
```suggestion
fmt/markdown:
	cd site
	../scripts/pnpm_install.sh
	pnpm exec markdown-table-formatter "../docs/**/*.md"
.PHONY: fmt/markdown
```

Reply:
Let us call it markdown. Your cal, its just a suggestion.

Reply:
@matifali , I'm leaning towards the specific `docs-md-tables` because I want to try getting [markdown linter to work as a formatter](https://megalinter.io/8.3.0/descriptors/markdown_markdownlint/) where it can, and I feel like that'll give us a way to debug if something fails. not super attached to that though

Reply:
not sure how to fix that `unstaged files` thing though: [error](https://github.com/coder/coder/actions/runs/12300639422/job/34329348747#step:12:1)

Reply:
Its about adding the table formatting in related gen make commands.

Tables are generated which are unformmated they are not matching the formatted tables.